src/query/config/m3query-dev-etcd.yml

```diff
@@ -66,3 +66,6 @@ limits:
     maxFetchedDatapoints: 5000000
   global:
     maxFetchedDatapoints: 500000000
+
```

Comment:
Should we update any of the other `yaml`/config files?

Reply:
Actually I like the idea of having it on by default; any chance we could get it into start_m3.sh as well?

Reply:
That would be pretty cool :)

Not sure how much effort that would entail; if it's a larger task maybe just throw up an Issue for it and leave it for future work?